test/unit/Fixture/ExampleClass.php

```diff
@@ -30,6 +30,9 @@ public function __construct()
         {
         }
 
+        /**
+         * @return int
```

Comment:
I suggest using a separate class. This asset is already used in other tests, and I indeed see some differences in other tests because of this.